packages/payment-experience-api/src/lib/paytrail.ts

```diff
@@ -1,4 +1,6 @@
-export const parseOrderIdFromPaytrailRedirect = (p: { query: any }) => {
+export const parseUuidFromPaytrailRedirectCheckoutStamp = (p: {
```

Comment:
Change uuid to orderId because we need to know the context, uuid can be like orderid/refund etc. Its harder to know later on when we touch this code what does this uuid mean in this context. 

You can create wrapper functions to make this usable to orderId and refundId but with same functionality. 

parseOrderIdFromPaytrailRedirect(){

}

parseRefundFromPaytrailRedirect(){

}

Reply:
And checkout-stamp is always the orderId when we create new payment/refund

Reply:
Sure, I'll add the wrapper functions to make the context clearer. 

The checkout-stamp on refunds will hold refundPaymentId, so the uuid to extract from refunds checkoust-stamp is refundId. checkout-refund-reference will hold the orderId.